src/common/isolatable_hardwares.cpp

```diff
@@ -647,6 +648,10 @@ std::optional<struct pdbg_target*>
             devTreeTgt = pdbg_target_parent("ocmb", devTreeTgt);
         }
 
+        /**
+         * Though ocmb has Location Code available now we need to use dimm
+         * location code in case of bonnell.
+         */
```

Comment:
Apart from the above comment what is changed in the executing code here?

Reply:
Removed the condition check for dimmCount>1

Reply:
Got it. Didn't notice. Apologies.